pkg/es/config/config.go

```diff
@@ -15,43 +15,72 @@
 package config
 
 import (
+	"bytes"
+	"context"
 	"time"
 
 	"github.com/pkg/errors"
+	"go.uber.org/zap"
 	"gopkg.in/olivere/elastic.v5"
 
 	"github.com/jaegertracing/jaeger/pkg/es"
 )
 
 // Configuration describes the configuration properties needed to connect to an ElasticSearch cluster
 type Configuration struct {
-	Servers     []string
-	Username    string
-	Password    string
-	Sniffer     bool          // https://github.com/olivere/elastic/wiki/Sniffing
-	MaxSpanAge  time.Duration `yaml:"max_span_age"` // configures the maximum lookback on span reads
-	NumShards   int64         `yaml:"shards"`
-	NumReplicas int64         `yaml:"replicas"`
+	Servers           []string
+	Username          string
+	Password          string
+	Sniffer           bool          // https://github.com/olivere/elastic/wiki/Sniffing
+	MaxSpanAge        time.Duration `yaml:"max_span_age"` // configures the maximum lookback on span reads
+	NumShards         int64         `yaml:"shards"`
+	NumReplicas       int64         `yaml:"replicas"`
+	BulkSize          int
+	BulkWorkers       int
+	BulkActions       int
+	BulkFlushInterval time.Duration
```

Comment:
do we want to add yaml tags for these? Because they are multi-word names, they will look differently otherwise

Reply:
where are these tags used?